cmd/apps/helm_operator_app.go

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+	"golang.org/x/mod/semver"
+)
+
+func MakeInstallHelmOperator() *cobra.Command {
+	var helmOperator = &cobra.Command{
+		Use:          "helm-operator",
+		Short:        "Install helm-operator",
+		Long:         "Install helm-operator",
+		Example:      "arkade install helm-operator --namespace default",
+		SilenceUsage: true,
+	}
+
+	helmOperator.Flags().StringP("namespace", "n", "default", "The namespace used for installation")
+	helmOperator.Flags().Bool("update-repo", true, "Update the helm repo")
+	helmOperator.Flags().Bool("helm3", true, "Use helm3, if set to false uses helm2")
```

Comment:
just spotted helm2. We no longer support helm2 in arkade, can this be taken out

Reply:
Removed